src/test/ui/parser/issues/issue-93282.stderr

```diff
@@ -3,6 +3,11 @@ error: expected `while`, `for`, `loop` or `{` after a label
    |
 LL |     f<'a,>
    |         ^ expected `while`, `for`, `loop` or `{` after a label
+   |
+help: add `'` to close the char literal
+   |
+LL |     f<'a',>
```

Comment:
This is somewhat unfortunate, but teaching the parser to avoid suggesting this is just as hard as solving the turbofish problem I think, because at this point we're parsing `f < $EXPR`

Reply:
Hm, maybe I can avoid this if the prev token is `<`. Or maybe this suggestion is just nonsense and the PR needs to be scrapped.

Reply:
It's fine, this is best effort.